ReactiveCocoa/Swift/Signal.swift

```diff
@@ -142,6 +142,25 @@ public func map<T, U, E>(transform: T -> U)(signal: Signal<T, E>) -> Signal<U, E
 	}
 }
 
+// Maps each non-nil value in a signal to a new value.  Nils are passed through
+// to the new signal.
+public func map<T, U, E>(transform: T -> U)(signal: Signal<T?, E>) -> Signal<U?, E> {
+	return Signal<U?, E> { observer in
```

Comment:
This could be implemented very easily as 
```swift
let originalSignal: Signal<T?, E>
let newSignal: Signal<U?, E> = originalSignal.map {
  $0.map(transform)
}
```

So I'm not sure it's worth adding to `Signal` itself :)